build.sh

```diff
@@ -26,6 +26,12 @@ function usage {
   exit 1
 }
 
+headline(){
+  echo -e "\e[1;34m#################################################################"
+  echo -e "##### $1 \e[1;37m"
+  echo -e "\e[1;34m#################################################################\e[0m"
```

Comment:
Can't remember if I commented on this in the previous PR, but can the ansi escapes here be handled using e.g. tput, or otherwise avoid breaking dumb terminals?

```bash
termblack=$(tput setaf 0)
termred=$(tput setaf 1)
termgreen=$(tput setaf 2)
termyellow=$(tput setaf 3)
termblue=$(tput setaf 4)
termpurple=$(tput setaf 5)
termaqua=$(tput setaf 6)
termreset=$(tput sgr0)
```

Reply:
Good point, I've ran into issues when running the scripts with Docker. It does not have any tty: `tput: No value for $TERM and no -T specified`. For now I've removed the colors since it doesn't really add much value.